docs/semgrep-ci/configuration-reference.md

```diff
@@ -21,12 +21,14 @@ SEMGREP_RULES="p/security-audit p/secrets"
 
 ## Suppressing blocking findings or errors
 
-Most CI providers block pull requests (PRs) or merge requests (MRs) on non-zero exit codes. Semgrep exits with a non-zero exit code (exit code 1) when it reports a blocking finding or there is an issue in running Semgrep (exit code 2 and above). For more information about specific Semgrep exit codes see [CLI reference](../../cli-reference/#exit-codes).
+Most CI providers block pull requests (PRs) or merge requests (MRs) on non-zero exit codes. Semgrep can be configured to suppress errors (this is the default behavior) or surface all errors depeding on the needs of your specific projects. This behavior is controlled via the `--suppress-errors/--no-suppress-errors` flag, and defaults to suppressing all errors _except_ blocking findings, which surface as exit code 1.
```

Comment:
```suggestion
Most CI providers block pull requests (PRs) or merge requests (MRs) on non-zero exit codes. Semgrep can be configured to suppress errors (this is the default behavior) or surface all errors depending on the needs of your teams and projects. Configure this behavior by the `--suppress-errors` or `--no-suppress-errors` options.
```

Reply:
Are they called options in YAML still? @spencerdrak I am not sure if we should call them flags... options sounds like configuration option... but yeah, `semgrep ci --suppress-errors` is a flag? Nathan made this suggestion, maybe you can help me to understand it better. :D https://www.notion.so/r2cdev/r2c-technical-communication-style-guide-4d1104e78927460aa8c7624aedd6561f#b859f45cf6ab4f559f8ee5482778a335

Reply:
It is a flag per this doc, will adjust

Reply:
@spencerdrak I was not really sure, we use double hyphens here. See:  `in the CLI these exclusively begin with either a single or double hyphen, options that begin with a single hyphen are short forms of an option that begins with a double hyphen; note that many options can be composed of multiple tokens.`

Reply:
```suggestion
Most CI providers block pull requests (PRs) or merge requests (MRs) on non-zero exit codes. Semgrep can be configured to suppress errors (this is the default behavior) or surface all errors depending on the needs of your teams and projects. Configure this behavior by the `--suppress-errors` or `--no-suppress-errors` flags.
```